src/lib/Microsoft.Fx.Portability.Reports.Html/HtmlHelper.cs

```diff
@@ -31,7 +36,7 @@ public static IHtmlContent TargetSupportCell<T>(this IHtmlHelper<T> helper, Targ
             var title = supported ? LocalizedStrings.Supported : LocalizedStrings.NotSupported;
             var icon = $"<svg class=\"support-icon\"><use href=#{imageId}></use></svg>";
 
-            return helper.Raw(Invariant($"<td class=\"textCentered\" title=\"{title}\">{icon}</td>"));
+            return helper.Raw(Invariant($"<td style=\"{style}\" class=\"textCentered\" title=\"{title}\">{icon} </td>"));
```

Comment:
I would set the class to targetSupportCell or something, and then define the styles in the style sheet.